integration-tests/src/test/scala/tech/beshu/ror/integration/suites/DataStreamApiSuite.scala

```diff
@@ -512,6 +825,16 @@ trait DataStreamApiSuite extends AnyFreeSpec
     }
   }
 
+  private def dataStreamBackingIndices(streamName: String): List[String] = {
+    adminDataStreamManager
+      .getDataStream(streamName)
+      .force()
+      .responseJson("data_streams").arr
```

Comment:
DataStreamManager#getDataStream could return GetDataStreamResult which could expose it as a method (see DocumentManager impl)

Reply:
oh, I thought we will have a method that will return a list of indices (strings)